exercises/rock_paper_scissors_lizard_spock.livemd

```diff
@@ -60,6 +60,37 @@ RockPaperScissorsLizardSpock.play(:lizard, :rock) # "Player 1 Wins!"
 RockPaperScissorsLizardSpock.play(:lizard, :lizard) # "Draw!"
 ```
 
+<details style="background-color: lightgreen; padding: 1rem; margin: 1rem 0;">
+<summary>Example Solution</summary>
+
+```elixir
```

Comment:
Wonderful!

If you'd like to submit a second example solution, there's a great version using the `in` keyword. We'll definitely keep the `case` solution though since it shows students how to use different control flow structures.

```elixir
{guess1, guess2} in [{:paper, :rock}, {:scissors, :paper}, etc]
```

I would make a `@winners` module attribute with a list of the winning tuple pairs, then simply write:

```elixir
{guess1, guess2} in @winners
```